packages/react-native-sdk/src/utils/StreamVideoRN/index.ts

```diff
@@ -67,10 +65,6 @@ export class StreamVideoRN {
       return;
     }
     this.config.push = pushConfig;
-    // After getting the config we should setup callkeep events, firebase handler asap to handle incoming calls from a dead state
-    setupFirebaseHandlerAndroid(pushConfig);
-    // setup ios handler for non-voip push notifications asap
-    setupRemoteNotificationsHandleriOS(pushConfig);
```

Comment:
main effect is here..

SDK will not automatically add listeners, integrators need to add them